libs/wingsdk/src/core/resource.ts

```diff
@@ -154,26 +154,30 @@ export abstract class Resource
       if (field.startsWith("this.")) {
         const key = field.substring(5);
         const obj: Resource = (this as any)[key];
-        if (!obj) {
+        if (obj === undefined) {
           throw new Error(
             `Resource ${this.node.path} does not have field ${key}`
           );
         }
-        if (!("_bind" in obj)) {
-          throw new Error(
-            `Resource ${this.node.path} field ${key} does not have a bind method`
-          );
-        }
```

Comment:
Since you refactored so we should only have to call `_bind` on resources, can we add this safety check back?

Reply:
Done